e2e/ansible/playbooks/hyperconverged/test-k8s-percona-mysql-pod/k8s-percona-pod-prerequisites.yml

```diff
@@ -29,6 +29,14 @@
   delegate_to: "{{ groups['kubernetes-kubeminions'].0 }}"
   become: true
 
+- name: Get $HOME of K8s master for kubernetes user
+  shell: source ~/.profile; echo $HOME
+  args:
+    executable: /bin/bash
+  register: result_kube_home
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+  changed_when: true
+
```

Comment:
There is an identical task in the same pre-req file which uses "/tmp" as the location for storing files (that too is stored as result_kube_home). Can you freeze upon this